### PR TITLE
Added new option fromSource for systempolicy discover

### PR DESCRIPTION
**Pull request by seswarrajan**
Dependent PR : https://github.com/accuknox/auto-policy-discovery/pull/455

Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

File: discover/discover.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ type Options struct {
 	Namespace   string
 	Clustername string
 	Labels      string
+	Fromsource  string
```
Comment (Ankurk99) [inline discover/discover.go:29]: FromSource  string

Comment (seswarrajan) [reply discover/discover.go:29]: The naming convention is **Fromsource**. 
Need to use the same.

Comment (daemon1024) [reply discover/discover.go:29]: I would prefer both of em to be updated to `FromSource` unless there's some other limitation

Comment (daemon1024) [reply discover/discover.go:29]: > But we are maintaining this naming convention for all other parameters as well

Only in the worker protobuf in which these fields were added recently. In general the type system in autopol seems to be camelCase only. But yeah it's fine if we want to keep following this.

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**